cmd/nitro/nitro.go

```diff
@@ -380,7 +380,11 @@ func mainImpl() int {
 		nodeConfig.Node.TxLookupLimit = 0
 	}
 
-	resourcemanager.Init(&nodeConfig.Node.ResourceMgmt)
+	err = resourcemanager.Init(&nodeConfig.Node.ResourceMgmt)
+	if err != nil {
+		flag.Usage()
+		log.Crit("failed to start resource management module")
```

Comment:
We should include the actual error as well, otherwise it just says that resource manager module failed without much context.

Also, nit: fully displayed messages should typically be capitalized [0].

[0] https://google.github.io/styleguide/go/decisions#error-strings

Reply:
Fixed, thanks.